doc/sphinx/source/users-guide/locality/onClauses.rst

```diff
@@ -0,0 +1,111 @@
+:title: Users Guide: On-Clauses
+
+.. default-domain:: chpl
+
+On-Clauses
+==========
+
+As mentioned previously, all Chapel programs begin execution as a
+single task running on locale #0.  Tasks can execute on remote locales
```

Comment:
Thinking about this a bit more based on your comment, I may actually end up refactoring this data-driven on-clauses section into the next page (which will show where variables are allocated as on-clauses move around and the PGAS nature of Chapel) to avoid the forward reference and need for a degenerate on-clause like this, but will leave it here until I get there...